examples/postgresql-multiple-datasources/README.md

```diff
@@ -0,0 +1,114 @@
+# Multiple PostgreSQL datasource example
+
+In this example we are provisioning a WildFly server containing a driver and multiple datasources for PostgreSQL databases.
+
+# WildFly Maven plugin configuration
+High level view of the WildFly Maven plugin configuration
+
+## Galleon feature-packs
+
+* `org.wildfly:wildfly-galleon-pack`
+* `org.wildfly.cloud:wildfly-cloud-galleon-pack`
+* `org.wildfly:wildfly-datasources-galleon-pack`
+
+## Galleon layers
+
+* `cloud-server`
+* `postgresql-driver`
+
+## CLI scripts
+WildFly CLI scripts executed at packaging time
+
+* None
+
+## Extra content
+Extra content packaged inside the provisioned server
+
+* None
+
+# Openshift build and deployment
+Technologies required to build and deploy this example
+
+* Helm chart for WildFly `wildfly/wildfly`. Minimal version `2.0.0`.
+
+# WildFly image API
+Environment variables from the [WildFly image API](https://github.com/wildfly/wildfly-cekit-modules/blob/main/jboss/container/wildfly/run/api/module.yaml) that must be set in the OpenShift deployment environment
+
+* NONE
+
+# WildFly cloud feature-pack added features
+
```

Comment:
The env variables related to datasource configuration in the helm chart file (eg: DB_SERVICE_PREFIX_MAPPING, PREFIX_JNDI, ... ) should be listed there. Yo ucould add a link to the env variable doc: https://github.com/wildfly/wildfly-cekit-modules/blob/main/jboss/container/wildfly/launch/datasources/module.yaml

Reply:
Added information about environment variables and linked to official documentation.